scripts/start.sh

```diff
@@ -18,7 +18,7 @@
 
 # INSTRUCTIONS:
 #
-# This script starts up a development server running Oppia's Foundation Website. 
+# This script starts up a development server running Oppia's Foundation Website.
 # It starts up a local Python SimplyHTTPServer.
```

Comment:
NB: should be SimpleHTTPServer (not Simply).

Reply:
Looks like this is a left-over comment from the pre-GAE time period. I have updated this docstring.